Gems/Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/ParallaxInput.azsli

```diff
@@ -32,7 +32,7 @@ void GetParallaxInput(float3 normal, float3 tangent, float3 bitangent, float hei
     if(o_parallax_feature_enabled)
     {
         float3 dirToCamera;
-        if(ViewSrg::m_projectionMatrix[0].w)
+        if(ViewSrg::m_projectionMatrix[3].w == 1)
```

Comment:
nit: I'd prefer `> 0` rather than `== 1` just for float testing safety.

Reply:
+1. Try to refrain from direct comparisons when it comes to floats.